src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +127,20 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void helpTest() {
```

Comment:
Actually it's a really small change, how about I leave this and you get a free `forFirstTimers` issue? I can help create that issue after this PR gets merged.